keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -199,59 +213,66 @@ process (see also: [Risks](#risks-and-mitigations)).
 
 ### Risks and Mitigations
 
-- *Risk*: We don't find a proper way to share secrets like the signing key  
-  *Mitigation*: TBA
-- *Risk*: Building all the packages for all the distributions and their version takes too long to be done nightly or via cutting the release  
-  *Mitigation*: TBA
+- _Risk_: We don't find a proper way to share secrets like the signing key
+  _Mitigation_: Using a third party tool like 1Password
+- _Risk_: Building all the packages for all the distributions and their version takes too long to be done nightly or via cutting the release  
+  _Mitigation_: We do not deliver nightly packages.
 
 ## Design Details
 
 ### Test Plan
 
-There should be a post-publish tests, which can be run as part or after the release process
-- pull packages from the  official mirrors (via the [redirector] if in place)
+There should be post-publish tests, which can be run as part or after the release process
+
+- pull packages from the official mirrors
 - assert that all the packages we expect to be published are actually published
+- assert that all publsihed packages have the expected checksums
 - assert that the packages and the repo metadata is signed with the current signing key
+- assert that installed packages contain the correct binary versions
 
 ### Graduation Criteria
 
-In general we can keep the current way of publishing (via googlers onto google's infrastructure) and introduce new infrastructure in parallel.
+In general we can keep the current way of publishing (via googlers onto Google's infrastructure) and introduce new infrastructure in parallel.
```

Comment:
IMHO: we should keep *publishing* but builds should be shifted to community control instead of Googler workstations. If that's the intention, I don't see it explicitly called out.

The existing signing+publishing can easily be switched to consume community built packages, they're distinct steps / tools in the scripts.

Reply:
This depends on which technical solution we choose. I added this paragraph to make it more clear that we can iteratively move forward: https://github.com/kubernetes/enhancements/pull/3434/files#diff-9356b6835a7626094e8636d5f5a8a1d71488e4d2abb948557ce41df3b77f88a1R204-R207

I think in the end it should be overall community owned, which would leave the current statement untouched, right?

> In general we can keep the current way of publishing (via googlers onto Google's infrastructure) and introduce new infrastructure in parallel.

Reply:
https://github.com/kubernetes/release/issues/913#issuecomment-1276364097 is also worth considering as an immediate stopgap, KEP or otherwise.

Reply:
I added one additional commit on top of the PR to explicitly mention the parallel improvement path. I'll make another comment about https://github.com/kubernetes/release/issues/913#issuecomment-1276364097 on the corresponding issue.